src/main/java/tr/com/srdc/cda2fhir/transform/DataTypesTransformerImpl.java

```diff
@@ -587,6 +587,7 @@ public Range tIVL_PQ2Range(IVL_PQ ivlpq) {
 		if (ivlpq.getLow() == null && ivlpq.getHigh() == null && ivlpq.getValue() != null) {
 			SimpleQuantity low = new SimpleQuantity();
 			low.setValue(ivlpq.getValue());
+			low.setSystem(vst.tOid2Url("2.16.840.1.113883.1.11.12839"));
 			rangeDt.setLow(low);
```

Comment:
The other branch cases don't need it because they use the method `tPQ2SimpleQuantity` and that method sets the systems for the high and low quantities. The method `tPQ2SimpleQuantity` takes in a `PQ` var and `ivlpq.getValue()` returns a type `BigDecimal` so I would have to convert `BigDecimal` to `PQ` type if I want to use the method used by the other branches. I can certainly do that but what I need was the simples solution. This is used for the `rate` field in `Dosage` and the system is need for the tests.

Reply:
Got it. But looking at this more closely now we should use getUnit() and getTranslations() as well to be equivalent to tPQ2SimpleQuantity. You might want to try getValue to tPQ2SimpleQuantity. I am not sure the relation between IVL_PQ and IVLB_PQ but I am assuming both extend PQ?